src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1679,7 +1679,10 @@ static SequenceComparison createSequenceComparison(final VariantContext variant,
                         sequenceComparison.getCodingSequenceAlleleStart(),
                         sequenceComparison.getAlignedCodingSequenceAlleleStart(),
                         correctedCodingSequence,
-                        sequenceComparison.getStrand()
+                        sequenceComparison.getStrand(),
+                        // Figure out if we are in a mitochondrial contig:
+                        // TODO: Make this more robust by detecting the mito contig based on the reference used.
```

Comment:
Either file an issue or address here.  Given that this is to add testing, I would probably do it now, if you think this is necessary.

Reply:
This isn't to add testing.  This is to use a more robust method of determining whether the contig is the mitochondrial contig for the particular reference you're using.

I've added it as #5364